roles/pid_generator/files/gen_pids.py

```diff
@@ -224,18 +265,26 @@ def openLocalPidDB():
 source_csv_url = SOURCE_CSV_URL
 urn_path = URN_PATH
 # The file containing the unmodified PID file (copied from Github)
+# useful for quickly reverse matching PIDs.
 raw_filename = RAW_FILENAME
+raw_target_name = urn_path + "/" + raw_filename
+
+# The auth token to access the source file in Github
 auth_token = AUTH_TOKEN
 
-# the URN xml for harvesting
-urn_target_xml_name = urn_path + "/urn_nbn_fi_lb.xml"
-raw_target_name = urn_path + "/" + raw_filename
+# name and location of the URN xml for harvesting by NLF's URN service
+urn_xml_filename = URN_XML_FILENAME
+urn_target_xml_name = urn_path + "/" + urn_xml_filename
 
+debug_email_address = DEBUG_EMAIL_ADDRESS
```

Comment:
The debug email also seems to be "read only", so we don't need this assignment either. Keeping to the original allcaps version also highlights the fact that it is a constant value from a config file.